docker-compose-services/postgres/README.md

```diff
@@ -52,6 +52,52 @@ hooks:
 There are also another non-plain-text formats that `pg_dump` can generate, and you might need to work with them. If that's the case, there is also
 a `ddev pg_restore` command that will restore `.ddev/pgsql-db/postgresql.db.dump` into `db`.
 
+## Multiple databases
+
+* By default, DDEV adds a single database called `db`. But it's possible to create additional databases when you first initialize this service.
```

Comment:
```suggestion
* By default, this recipe adds a single database called `db`. But it's possible to create additional databases when you first initialize this service.
```

Reply:
Good suggestion. I keep forgetting I'm not in the "supported" DDEV repo.